modules/transdecoder/longorf/meta.yml

```diff
@@ -0,0 +1,62 @@
+name: transdecoder_longorf
+description: TransDecoder itentifies candidate coding regions within transcript sequences. it is used to build gff file.
+keywords:
+  - eucaryotes
+  - gff
+tools:
+  - transdecoder:
+      description: TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.
+      homepage: https://github.com/TransDecoder
+      documentation: https://github.com/TransDecoder/TransDecoder/wiki
+      tool_dev_url: https://github.com/TransDecoder/TransDecoder
+      doi: ""
+      licence: ['Broad Institute']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fasta:
+      type: file
+      description: fasta file
+      pattern: "*.{fasta}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - pep:
+      type: file
+      description: all ORFs meeting the minimum length criteria, regardless of coding potential. file
+      pattern: "*.{pep}"
+  - gff3:
+      type: file
+      description: positions of all ORFs as found in the target transcripts. file
```

Comment:
The format is obvious from the name of the channel, but maybe good to spell out.